Integrations/Akamai_WAF/Akamai_WAF.yml

```diff
@@ -0,0 +1,270 @@
+      description: Number of element in the list
+      type: Number
+    - contextPath: Akamai.NetworkLists.List.Elements
+      description: Elements in the lisy
+      type: String
+  - arguments:
+    - default: false
+      description: Network list ID
+      isArray: false
+      name: network_list_id
+      required: true
+      secret: false
+    deprecated: false
+    description: Delete network list
+    execution: false
+    name: akamai-delete-network-list
+  - arguments:
+    - default: false
+      description: Network list ID
+      isArray: false
+      name: network_list_id
+      required: true
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Enviorment to activate the network list
+      isArray: false
+      name: env
+      predefined:
+      - STAGING
+      - PRODUCTION
+      required: true
+      secret: false
+    - default: false
+      description: Comment to be logged
+      isArray: false
+      name: comment
+      required: false
+      secret: false
+    - default: false
+      description: List of emails seprated with commas
+      isArray: true
+      name: notify
+      required: false
+      secret: false
+    deprecated: false
+    description: Activate network list on Staging or Production
+    execution: false
+    name: akamai-activate-network-list
+  - arguments:
+    - default: false
+      description: Network list ID
+      isArray: false
+      name: network_list_id
+      required: true
+      secret: false
+    - default: false
+      description: War-room entry ID of sample file
+      isArray: false
+      name: entry_id
+      required: false
+      secret: false
+    - default: false
+      description: Elements to be added, comma seprated
+      isArray: true
+      name: elements
+      required: false
+      secret: false
+    deprecated: false
+    description: Add elements to network list
+    execution: false
+    name: akamai-add-elements-to-network-list
+  - arguments:
+    - default: false
+      description: Network list ID
+      isArray: false
+      name: network_list_id
+      required: true
+      secret: false
+    - default: false
+      description: Element to be removed
+      isArray: false
+      name: element
+      required: true
+      secret: false
+    deprecated: false
+    description: Remove element from network list
+    execution: false
+    name: akamai-remove-element-from-network-list
+  - arguments:
+    - default: false
+      description: Network list ID
+      isArray: false
+      name: network_list_id
+      required: true
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Enviorment Produciton or Staginf
+      isArray: false
+      name: env
+      predefined:
+      - PRODUCTION
+      - STAGING
+      required: true
+      secret: false
+    deprecated: false
+    description: Get network list activation status in Production or staging
```

Comment:
@GalRabin please implement a Generic Polling playbook for activation

Reply:
Ok